pkg/providers/tinkerbell/pbnj/client.go

```diff
@@ -0,0 +1,60 @@
+package pbnj
+
+import (
+	"context"
+	"os"
+
+	pbnjClient "github.com/tinkerbell/cluster-api-provider-tinkerbell/pbnj/client"
+	pbnjv1 "github.com/tinkerbell/pbnj/api/v1"
+
```

Comment:
nit: I think we can remove the names, `pbnjClient` and `pbnjv1` for the imports.
Names make sense when there are imports with the same name. 
Here we could directly do `client.` and `v1.`